test/system/admin/members_test.rb

```diff
@@ -0,0 +1,38 @@
+require "application_system_test_case"
+
+class Admin::MembersTest < ApplicationSystemTestCase
+  include ActionView::Helpers::NumberHelper
+
+  def setup
+    @member = create(:verified_member_with_membership, :with_pronunciation)
+    sign_in_as_admin
+  end
+
+  test "admin can view member's profile" do
```

Comment:
These tests are great! Thanks for adding this coverage to help us catch future regressions in this part of the app.